src/fidesctl/api/main.py

```diff
@@ -65,16 +65,26 @@ def configure_routes() -> None:
 app.dependency_overrides[lib_verify_oauth_client] = verify_oauth_client
 
 
-@app.on_event("startup")
-async def create_webapp_dir_if_not_exists() -> None:
-    """Creates the webapp directory if it doesn't exist."""
+def get_ui_file(path: Path) -> Optional[Path]:
+    """There are 3 cases to handle when trying to get index.html
+    1. We are in a package environment so need to get the index.html that is packaged
```

Comment:
is there a way to get the packaged code and the dev env code to live in the same place? That way we don't have an extra check and it follows the `dev like you prod` mantra. I might be misunderstanding the complexity here though!

Reply:
yeah I think there is, though I haven't tried in the package environment yet. at first I thought there wasn't, hence the comment, but I'm hopeful and will update the comment accordingly :)